### ISSUE TITLE
sound@cinnamon.org applet - Make the 100% mark really magnetic

### ISSUE THREAD

**Issue by claudiux** [linuxmint/Cinnamon#7375]
```
 * Cinnamon version : 3.6.7, built with mint-dev-tools
 * Distribution : Mint 18.3
 * 64 bit
 ```

**Issue**
(With the volume max set to 150% in sound settings.)
When the volume slider is on the 100% mark, the real volume is nearly of 96%.

**Steps to reproduce**
Click on the icon of the sound applet, drag the slider to 88% then scroll up one time.

**Expected behaviour**
Set the real volume to 100%, as the slider is.

**Other information**
Proposed solution : 
In the _update function, replace the line:
let value, visible_value, delta = VOLUME_ADJUSTMENT_STEP;
by the line:
let value, visible_value, delta = VOLUME_ADJUSTMENT_STEP * this.applet._volumeMax / this.applet._volumeNorm;

**Comment by claudiux** [comment]
(With the volume max set to 150% in sound settings.)

**Comment by claudiux** [comment]
This issue is solved by PR #7376.

**Comment by claudiux** [comment]
I close this issue because PR #7376 is now merged.
